src/obfl-specification.html

```diff
@@ -2207,6 +2273,14 @@ <h4 id="L9512" class="include">on-page-end</h4>
     <dd>list-of-references</dd>
   <dt>Attributes</dt>
     <dd>None</dd>
+  <dt>Variables</dt>
+    <dd><dl>
+        <dt>started-page-number</dt>
+          <dd>The variable holding the number of the started page. It can be
```

Comment:
"of the started page" -> "of the _ended_ page".

Reply:
Done.